mcslock/tests.c

```diff
@@ -4,7 +4,7 @@
 
 int main(void)
 {
-    mcslock_t lock;
+    _Atomic(mcslock_t) lock;
```

Comment:
It is a bit inconvenient. Users may regard `mcslock_t` as the userspace implementation of MCS lock and use `mcslock_t` without `_Atomic`.

Reply:
I moved definition of _Atomic for lock from test.c to msclock.h. Currently, users can use `mcslock_t` without `_Atomic`